src/components/Dashboard/FilingHistoryList/HeaderActions.vue

```diff
@@ -131,9 +132,9 @@ export default class HeaderActions extends Mixins(AllowableActionsMixin) {
 
       case FilingTypes.CHANGE_OF_ADDRESS:
       case FilingTypes.CHANGE_OF_DIRECTORS:
-        if (this.isBenBcCccUlc) {
-          // correction via Edit UI if current type is BC, CC, ULC, or BEN
-          // To-Do for the future: Revisit this when we do Coop corrections in Edit UI
+      case FilingTypes.SPECIAL_RESOLUTION:
+        if (this.isBenBcCccUlc || (filing.name === FilingTypes.SPECIAL_RESOLUTION && this.isCoop)) {
+          // correction via Edit UI if current type is BC, CC, ULC, BEN or coop special resolution correction
```

Comment:
Did you see the comment on line 121? I think you also need to update something in Todo List to resume a draft SR correction.

Reply:
sorry I missed it, I just update Todo List, the resume a draft SR correction will redirect to edit ui now.

Reply:
Looks good, thanks.